install.sh

```diff
@@ -309,7 +329,7 @@ bye() {  # Prints a friendly good bye message and exits the script.
 
         echo "🔴 The containers didn't seem to start correctly. Please run the following command to check containers that may have errored out:"
         echo ""
-        echo -e "sudo docker-compose -f docker-compose.yml ps -a"
+        echo -e "docker-compose -f docker-compose.yml ps -a"
```

Comment:
will this `sudo ` break on `non-linux` environments ? I guess if the user didn't do `sudo usermod -aG docker $USER` to run as `$USER` ... 

If you are not sure, we could run different commands per OS